editor.planx.uk/src/@planx/components/shared/Preview/MoreInfo.tsx

```diff
@@ -14,33 +14,33 @@ interface StyledDrawerProps extends DrawerProps {
   drawerWidth: number;
 }
 
-const Root = styled(Drawer)<StyledDrawerProps>(
-  ({ theme, drawerWidth, open }) => ({
+const Root = styled(Drawer, {
+  shouldForwardProp: (prop) => prop !== "drawerWidth",
```

Comment:
Passing the prop `drawerWidth` into the styled `Root` component (line 66) automatically passes it down to the MUI Drawer component, which does not recognise the prop and throws an error in the console.

`shouldForwardProp` allows us to specify that we should not forward `drawerWidth` along to the underlying component.

Docs: https://mui.com/system/styled/#styled-component-options-styles-component